misc/docker_tests/base_images/fedora_21/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM fedora:21
+
+MAINTAINER Lion Krischer
+
+RUN rpm -Uvh http://download.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
+# Can fail on occasion.
+RUN yum -y upgrade || true
+RUN yum install -y numpy scipy gcc-gfortran python-matplotlib python-sqlalchemy python-lxml python-mock python-basemap python-basemap-data python-tornado python-pip
+RUN pip install flake8 suds-jurko future
+# Fedora has a slightly messed up basemap installation.
+RUN mkdir -p /usr/lib64/python2.7/site-packages/mpl_toolkits/basemap/data/
+RUN mv /usr/share/basemap/* /usr/lib64/python2.7/site-packages/mpl_toolkits/basemap/data/
```

Comment:
The fix for this went into testing on the 15, so it should be generally available around the 22/23 and we can remove this bit then.